community/index.md

```diff
@@ -15,6 +15,7 @@ title: Get in touch!
     </ul>
     <ul>
       <li><a href="https://webchat.freenode.net/?channels=jclouds">Chat</a> with us on IRC at <a href="https://botbot.me/freenode/jclouds/">#jclouds on Freenode</a></li>
+      <li>Join the <a href="https://the-asf.slackarchive.io/jclouds">#jclouds Slack channel</a> at the <a href="https://the-asf.slack.com/">ASF workspace</a>. Ask us for an invitation to join!</li>
```

Comment:
@nacx Is there a "sign up" link we could add? Or a `mailto:` URL with an appropriate subject we could construct?

Reply:
Not really. For non-apache committers access to the ASF workspace is by manual invitation only, and if anyone wants to get invited, I'd rather have them drop an email or say hi on the IRC instead of a default empty email with just a subject.